### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by VedanthB**
- added responsive top nav
- added side nav

https://user-images.githubusercontent.com/75097551/152412868-84d060a9-ac50-45e6-8d69-d2e655ff187f.mov

### CODE REVIEW

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  align-items: center;
+}
+/* nav .links li {
+  list-style: none;
+} */
+/* nav .links a {
+  display: block;
+  padding: 1em;
+  color: #fff;
+} */
```
Comment (stutisk) [inline components/navigation.css:36]: remove comments

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  transition: 0.5s;
+}
+
+/* Position and style the close button (top right corner) */
+
+.sidenav .closebtn {
+  position: absolute;
+  top: 60px;
+  right: 25px;
+  font-size: 36px;
```
Comment (stutisk) [inline components/navigation.css:117]: you can try using rem instead of px

File: components/navigation.css [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+.rating {
+  border: none;
+  float: left;
+}
+
+.rating > input {
+  display: none;
+}
+.rating > label:before {
```
Comment (stutisk) [inline components/navigation.css:144]: its ::before instead of :before

File: components/navigation.css [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  margin: 5px;
+  font-size: 1.25em;
+  font-family: FontAwesome;
+  display: inline-block;
+  content: "\f005";
+}
+
+.rating > .half:before {
```
Comment (stutisk) [inline components/navigation.css:152]: same here

**Review by stutisk — COMMENTED** [review]
else,code looks good to me

File: components/navigation.css (hunk 2 above)
Comment (VedanthB) [reply components/navigation.css:117]: okay will try using rem

File: components/navigation.css (hunk 3 above)
Comment (VedanthB) [reply components/navigation.css:144]: okay will make the changes

File: components/navigation.css [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+    width: auto;
+    justify-content: center;
+  }
+  nav .links {
+    float: none;
+    position: fixed;
+    z-index: 9;
+    left: 0;
+    right: 0;
+    top: 60px;
```
Comment (dev-enforced) [inline components/navigation.css:68]: You can use rem as mentioned by @stutisk earlier here!!

File: styles.css [hunk 6]
```diff
@@ -8,3 +8,8 @@
   margin: 10px;
   padding: 10px;
 }
+
+.test-2 {
+  margin: 50px;
```
Comment (dev-enforced) [inline styles.css:13]: Here too same as above!!

**Review by dev-enforced — CHANGES_REQUESTED** [review]
Apart from the other above mentioned changes ,code looks good to me !!
One small clarification to make: for navigation branch card component files are also present.Is it because the component branches haven't merged.

**Comment by VedanthB** [comment]
> Apart from the other above mentioned changes ,code looks good to me !! One small clarification to make: for navigation branch card component files are also present.Is it because the component branches haven't merged.

yes its because the branches have not been merged

**Pull request merged**